OpenRA.Mods.Common/Traits/Repairable.cs

```diff
@@ -122,7 +122,7 @@ void AfterReachActivities(Actor self, Order order, IMove movement)
 				self.QueueActivity(new CallFunc(() =>
 				{
 					self.SetTargetLine(Target.FromCell(self.World, rp.Location), Color.Green);
-					self.QueueActivity(movement.MoveTo(rp.Location, order.TargetActor));
+					self.QueueActivity(new Move(self, movement.NearestMoveableCell(rp.Location), WDist.FromCells(8)));
```

Comment:
You can't hardcode Move here, that only works for Mobile, so will break this for other `IMove` implementations.